internal/govulncheckshim/client.go

```diff
@@ -0,0 +1,110 @@
+package govulncheckshim
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/google/osv-scanner/pkg/models"
+	"golang.org/x/vuln/client"
+	gvcOSV "golang.org/x/vuln/osv"
+)
+
+type localSource struct {
+	vulnList         []models.Vulnerability
+	response         map[string]*models.Vulnerability
+	aliasToIDs       map[string][]*models.Vulnerability
+	moduleToIDs      map[string][]*models.Vulnerability
+	lastModifiedTime time.Time
+	client.Client
+}
+
+func newClient(vulns []models.Vulnerability) *localSource {
+	outLs := localSource{
```

Comment:
nit: it wasn't obvious what `outLs` meant. Can we just call this variable `client` instead?